addons/docs/src/frameworks/svelte/svelte-docgen-loader.ts

```diff
@@ -26,11 +55,10 @@ export default async function svelteDocgen(source: string) {
     // populate filename in docgen
     componentDoc.name = path.basename(file);
 
-    const componentName = path.parse(resource).name;
-
+    const componentName = get_name_from_filename(resource);
     docgen = `
-    ${componentName}.__docgen = ${JSON.stringify(componentDoc)};
-  `;
+${componentName}.__docgen = ${JSON.stringify(componentDoc)};
```

Comment:
Maybe use `ts-dedent` here if the indentation matters instead of manually dedenting in this code.